### ISSUE TITLE
Extremely sluggish/slow

### ISSUE THREAD

**Issue by NULLx76** [flathub/com.discordapp.Discord#61]
The flatpak version of discord is very sluggish for me and it takes more than a second to transition when clicking on buttons. It has become so bad that I had to resort to using the non-flatpak edition. 

There doesn't seem to be anything in the log, but I'll post it here anyways:
```
victor@eevee ~ % com.discordapp.Discord 
Gtk-Message: 22:57:42.031: Failed to load module "canberra-gtk-module"
Gtk-Message: 22:57:42.031: Failed to load module "canberra-gtk-module"
Discord 0.0.9
Starting app.
Starting updater.
[Modules] Modules initializing
[Modules] Distribution: remote
[Modules] Host updates: enabled
[Modules] Module updates: enabled
[Modules] Module install path: /home/victor/.var/app/com.discordapp.Discord/config/discord/0.0.9/modules
[Modules] Module installed file path: /home/victor/.var/app/com.discordapp.Discord/config/discord/0.0.9/modules/installed.json
[Modules] Module download path: /home/victor/.var/app/com.discordapp.Discord/config/discord/0.0.9/modules/pending
[Modules] No updates to install
[Modules] Checking for host updates.
[Modules] Host is up to date.
[Modules] Checking for module updates at https://discordapp.com/api/modules/stable/versions.json
[Modules] No module updates available.
[Modules] Starting to install discord_media...
[Modules] Fetching discord_media@0 from https://discordapp.com/api/modules/stable/discord_media/0
[Modules] Streaming discord_media@0 [251 bytes] to /home/victor/.var/app/com.discordapp.Discord/config/discord/0.0.9/modules/pending/discord_media-0.zip
[Modules] Finished module downloads. [success: 0] [failure: 1]
```

**Comment by tannisroot** [comment]
I have the same issue. It happened to me after my PC returned from sleep.

**Comment by Marzal** [comment]
Same issue here. The only warning I see is about **canberra-gtk-module**
I use Arch Linux with Gnome 3.32.1 and amdgpu open driver on RX 580 gpu

Xwayland seems to be a bit better (subjectively).

```
Gtk-Message: 12:17:04.340: Failed to load module "canberra-gtk-module"
Gtk-Message: 12:17:04.340: Failed to load module "canberra-gtk-module"
Discord 0.0.9
Starting app.
Starting updater.
[Modules] Modules initializing
[Modules] Distribution: remote
[Modules] Host updates: enabled
[Modules] Module updates: enabled
[Modules] Module install path: /home/dmc/.var/app/com.discordapp.Discord/config/discord/0.0.9/modules
[Modules] Module installed file path: /home/dmc/.var/app/com.discordapp.Discord/config/discord/0.0.9/modules/installed.json
[Modules] Module download path: /home/dmc/.var/app/com.discordapp.Discord/config/discord/0.0.9/modules/pending
[Modules] No updates to install
[Modules] Checking for host updates.
[Modules] Host is up to date.
[Modules] Checking for module updates at https://discordapp.com/api/modules/stable/versions.json
[Modules] No module updates available.
```

**Comment by WyRe** [comment]
I agree with you all; I'm suffering a really bad performance. Mainly with p2p calls.

**Comment by HarryMichal** [comment]
`canberra-gtk-module` is for alert sound. This error is not exclusive for Discord. So I wouldn't mark it as the cause.

When using https://extensions.gnome.org/extension/615/appindicator-support/ to get tray icons I noticed a large number of errors coming from it during a call. I'm not sure if they happen only during calls. But the performance of the app drops noticeably when using the extension.

**Comment by dtomvan** [comment]
Also applies to the arch packages `discord_arch_electron` and `discord`.

**Comment by dtomvan** [comment]
Fix: downgrade electron to version `9.4.4`. I'm not too familiar with Flatpak, but it works for me in Arch's pacman.

**Comment by guihkx** [comment]
This is an old issue, and there hasn't been any recent comments on it.

For that reason, I'll close this one. Feel free to re-open if the problem still exists.
